app/views/content/funding-and-support/scholarships-and-bursaries.md

```diff
@@ -75,7 +98,7 @@ The amount you receive depends on the year that you start your QTS course. You d
 
 ### Undergraduate bursary if you're former military personnel
 
-You may be eligible for a £40,000 tax-free undergraduate bursary for veterans if you’re:
+You may be eligible for a £40,000 tax-free [undergraduate bursary for veterans](/funding-and-support/if-youre-a-veteran) if you’re:
```

Comment:
is this link confusing? goes to the veterans page rather than information just about the bursary

would it be better to have another standalone link instead which says something like 'Learn more about support for veterans wanting to train to teach' (but better worded!)

Reply:
good point, shall amend